### PR TITLE
chore: [Card] support a11y

### PR DESCRIPTION
**Pull request by boomboomchen**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [ ] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- feat: Card 支持 A11y

---

🇺🇸 English
- feat: Card supports A11y

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### CODE REVIEW

File: content/show/card/index-en-US.md [hunk 1]
```diff
@@ -691,6 +699,10 @@ function Demo() {
 |style      |Meta style            |CSSProperties|-       |1.21.0 |
 |title      |title                 |ReactNode    |-       |1.21.0 |
 
+## Accessibility
+
+- Card is a container-type component, and any elements inside the card need to follow their respective accessibility guidelines.
+
```
Comment (shijiatongxue) [inline content/show/card/index-en-US.md:705]: 这里可以把 aria-busy 也写下

File: content/show/card/index-en-US.md [hunk 2]
```diff
@@ -470,14 +478,14 @@ function Demo() {
                     />
                 }
                 headerExtraContent={
-                    <Skeleton style={{width: 50}} placeholder={<Paragraph rows={1} />} loading={loading}>
+                    <Skeleton style={{width: 50}} placeholder={<Paragraph rows={1} />} loading={loading} aria-busy={loading}>
```
Comment (shijiatongxue) [inline content/show/card/index-en-US.md:481]: 这里 Skeleton 会接受这个属性不

File: content/show/card/index-en-US.md (hunk 1 above)
Comment (boomboomchen) [reply content/show/card/index-en-US.md:705]: 好的，我加一下

File: content/show/card/index-en-US.md (hunk 2 above)
Comment (boomboomchen) [reply content/show/card/index-en-US.md:481]: 可以接受，但是skeleton还没改造，会有ts报错

**Pull request closed without merging**